lib/builderator/interface/packer.rb

```diff
@@ -138,12 +138,20 @@ def _chef_provisioner_base
           :environments_path => Config.local.environment_path,
           :chef_environment => Config.profile.current.chef.environment,
           :json => Config.profile.current.chef.node_attrs,
+          :staging_directory => Config.chef.staging_directory,
+          :execute_command => _chef_execute_command(true),
+          :install_command => _chef_install_command(true)
```

Comment:
It's actually `:staging_directory` that does bad stuff. We can't guarantee a specific directory structure in a container and in this case it's easier to just let packer and chef do their things rather than specifying it.